README.md

```diff
@@ -93,4 +93,4 @@ Event website (MIAMI, December 2021): https://www.dcentralcon.com
 * William Penney III
 * Maciej Frik
 * Scott Andersen
```

Comment:
Dany Sanchez

Reply:
You have added a comment on top of somebody else's pull request. Please start again and create a new (your own) pull request with your change.